Headers/DebugServer2/Host/Linux/ARM64/Syscalls.h

```diff
@@ -34,6 +34,13 @@ static inline uint32_t MakeMovNegImmInstr(uint8_t reg, int32_t val) {
   return base | (static_cast<uint16_t>(-val - 1) << 5) | reg;
 }
 
+static inline uint32_t MakeLdrRelInstr(uint8_t reg, int32_t offset) {
+  DS2ASSERT(reg <= 31);
+  DS2ASSERT(offset % 4 == 0);
+  static const uint32_t base = 0x58000000;
+  return base | (static_cast<uint16_t>(offset / 4) << 5) | reg;
```

Comment:
This is interesting. I think we could simply do `offset << 3`, but I like the preservation of intent. The compiler also is probably smart enough to figure out this optimization anyway.